packages/browser-integration-tests/scripts/detectFlakyTests.ts

```diff
@@ -0,0 +1,53 @@
+import * as glob from 'glob';
+import * as path from 'path';
+import * as childProcess from 'child_process';
+import { promisify } from 'util';
+
+const exec = promisify(childProcess.exec);
+
+async function run(): Promise<void> {
+  const testPaths = getTestPaths();
+
+  const cwd = path.join(__dirname, '../');
+  const runCount = parseInt(process.env.TEST_RUN_COUNT || '10');
+
+  for (const testPath of testPaths) {
+    console.log(`Running test: ${testPath}`);
+    const start = Date.now();
+
+    try {
+      await exec(`yarn playwright test ${testPath} --browser='all' --repeat-each ${runCount}`, {
```

Comment:
Do we have to specify the paths here? Shouldn't it be enough to call `playwright test ./suites` to execute them all?

Reply:
I want to avoid running a single command that takes 60 minutes or so 😅 Also this makes it much easier to filter out all the successful stuff and only report on failing things.

Reply:
Ahh sorry, I missed the part with the changed tests. Makes sense!